### PR TITLE
fix typos and wording in ref-based RNASeq tutorial

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Spliced mappers have been developed to efficiently map transcript-derived reads
 
 Here, we will use HISAT2, a successor to TopHat2 that is faster with low memory requirements.
 
-To run efficiently the mapping, HISAT2 needs to know on important parameters about the sequencing library: the library type.
+To conduct the mapping efficiently, HISAT2 needs to know on important parameter about the sequencing library: the library type.
```
Comment (nsoranzo) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: s/on/one/ ?

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 2]
```diff
@@ -420,58 +421,58 @@ The recommended mode is "union", which counts overlaps even if a read only share
 >    >    <summary>Click to view answers</summary>
 >    >    <ol type="1">
 >    >    <li>The useful result file is a tabular file with two columns: the gene id and the number of reads mapped on the corresponding gene</li>
->    >    <li>To display the most found feature, we need first to sort the output file with the feature and the number of reads found for these feature. We do that using Sort tool, on the second column and in descending order. And we found that FBgn0017545 is the feature with the most reads mapped on it with 4,030 reads.</li>
+>    >    <li>To display the most abundantly detected feature, we need to sort the output file with the features and the number of reads mapped to them. When do this using the Sort tool on the second column and in descending order, we find that FBgn0017545 is the feature with the most reads (4,030) mapped on it.</li>
```
Comment (nsoranzo) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: I'd rephrase the last sentence as: `This can be done using the Sort tool on the second column and in descending order, which reveals that FBgn0017545 is the feature with the most reads (4,030) mapped on it.`

**Comment by wm75** [comment]
@nsoranzo: thanks for the very fast review and the good catches

**Comment by willdurand** [comment]
Thanks @wm75!
